config/locales/de/views/tasks.yml

```diff
@@ -0,0 +1,116 @@
+---
+de:
+  tasks:
+    access_request_mailer:
+      greeting: Hallo %{user}
+      message_line1: '%{user} möchte an Ihrer Aufgabe "%{task}" mitarbeiten.'
+      message_line2: Wenn Sie den/die Benutzer:in zum/zur Co-Autor:in machen wollen, oder in eine Gruppe einladen wollen, klicken sie auf den Link unten.
+      message_line3: Ansonsten können Sie diese E-Mail ignorieren.
+      subject: '%{user} möchte an Ihrer Aufgabe "%{task}" mitarbeiten.'
+    editor:
+      button:
+        extract_text: Text extrahieren
+        toggle_file: Datei-Upload / Editor umschalten
+        upload_a_new_file: Eine neue Datei hochladen
+    export_actions:
+      create_new: Neu erstellen
+      overwrite: Überschreiben
+    file_config:
+      configuration: Konfiguration
+      delayed: Verzögert
+      display: Anzeigen
+      download: Herunterladen
+    form:
+      button:
+        add_model_solution: Musterlösung hinzufügen
+        add_test: Test hinzufügen
+        remove_model_solution: Musterlösung entfernen
+        remove_test: Test entfernen
+        save_task: Aufgabe speichern
+      errors:
+        not_public_without_prog_lang: kann nicht öffentlich sein, solange die Programmiersprache nicht festgelegt ist
+        prog_lang_not_empty_while_public: darf nicht leer sein, wenn die Sichtbarkeit auf öffentlich eingestellt ist
+      no_license: Keine Lizenz
+      unchangeable_groups: Nicht änderbare Gruppen
+      visibility_warning: Diese Aufgabe befindet sich derzeit in einer Sammlung. Andere Benutzer:innen können Ihre Aufgabe nicht anzeigen oder bearbeiten. Der Titel bleibt jedoch für diese Benutzer:innen sichtbar.
+    import_actions:
+      button:
+        import_copy: Als Kopie importieren
+        overwrite: Überschreiben
+        show_task: Aufgabe anzeigen
+    import_dialog_content:
+      path: 'Pfad: %{path}'
+      task_with_uuid: 'Aufgaben-UUID: %{task_uuid}'
+    index:
+      button:
+        choose_file: Datei auswählen
+        import_tasks: Aufgaben importieren
+        import_zip: ZIP-Datei importieren
```

Comment:
In some German locales above, we used the form "Zip". I would suggest unifying that, potentially using "ZIP" (maybe check the English locales as well).